pallets/runtime/develop/src/runtime.rs

```diff
@@ -140,6 +140,11 @@ parameter_types! {
     // Portfolio:
     pub const MaxNumberOfFungibleMoves: u32 = 10;
     pub const MaxNumberOfNFTsMoves: u32 = 100;
+
+    // State trie Migration
+    pub const MigrationSignedDepositPerItem: Balance = 1_000;
+    pub const MigrationSignedDepositBase: Balance = 1_000_000;
+    pub const MaxKeyLen: u32 = 512;
```

Comment:
We should use a higher value for this 1024 or 2048.
Based on the comment here:
https://github.com/PolymeshAssociation/polkadot-sdk/blob/74ef745933703849d756dc16d41d938157d6943a/substrate/frame/state-trie-migration/src/lib.rs#L470